docs/automation/crmscript/reference/CRMSCript.Native.EventDataDialogDefinition.yml

```
@@ -0,0 +1,983 @@
+  name: EventDataDialogDefinition
+  nameWithType: EventDataDialogDefinition
+  fullName: CRMScript.Native.EventDataDialogDefinition
+  type: Class
+  summary: "\nUtility class to generate a dialog to use in CRMScript triggers.\n"
+  example: 
+- uid: CRMScript.Native.EventDataDialogDefinition.#ctor
+  commentId: M:CRMScript.Native.EventDataDialogDefinition.#ctor
+  id: '#ctor'
+  so.intellisense: Void.EventDataDialogDefinition
+  so.version: 10.1.9
+  langs:
+  - crmscript
+  name: EventDataDialogDefinition()
+  nameWithType: EventDataDialogDefinition.#ctor
+  fullName: CRMScript.Native.EventDataDialogDefinition.#ctor
+  type: Constructor
+  summary: "\nInitializes a new instance of the EventDataDialogDefinition class.\n"
+  remarks: 
+  syntax: 
+    content: EventDataDialogDefinition
+  example: 
+- uid: CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String)
+  commentId: M:CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String)
+  id: 'addCheckbox(String,String)'
+  so.intellisense: EventDataDialogDefinition.addCheckbox(String,String)
+  so.version: 10.1.9
+  langs:
+  - crmscript
+  name: addCheckbox(String,String)
+  nameWithType: EventDataDialogDefinition.addCheckbox(String,String)
+  fullName: CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String)
+  type: Method
+  summary: "\nAdd a checkbox field with given label.\n"
+  remarks: 
+  syntax: 
+    content: Void addCheckbox(String id, String label)
+    parameters:
+    - id: id
+      type: CRMScript.Global.String
+      description: 
+    - id: label
+      type: CRMScript.Global.String
+      description: 
+    return:
+      type: CRMScript.Global.Void
+  example: "\n<pre><code>EventDataDialogDefinition def;\ndef.addCheckbox(&quot;checkme&quot;, &quot;My checkbox&quot;);</code></pre>\n"
+- uid: CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String,Bool)
+  commentId: M:CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String,Bool)
+  id: 'addCheckbox(String,String,Bool)'
+  so.intellisense: EventDataDialogDefinition.addCheckbox(String,String,Bool)
+  so.version: 10.1.9
+  langs:
+  - crmscript
+  name: addCheckbox(String,String,Bool)
+  nameWithType: EventDataDialogDefinition.addCheckbox(String,String,Bool)
+  fullName: CRMScript.Native.EventDataDialogDefinition.addCheckbox(String,String,Bool)
+  type: Method
+  summary: "\nAdd a checkbox field with given label. Possible to set field as mandatory.\n"
+  remarks: 
+  syntax: 
+    content: Void addCheckbox(String id, String label, Bool mandatory)
+    parameters:
+    - id: id
+      type: CRMScript.Global.String
+      description: 
+    - id: label
+      type: CRMScript.Global.String
+      description: 
+    - id: mandatory
+    type: CRMScript.Global.Bool
+    description: 
+    return:
+      type: CRMScript.Global.Void
+  example: "\n<pre><code>EventDataDialogDefinition def;\ndef.addCheckbox(&quot;checkme&quot;, &quot;My checkbox&quot;,true);</code></pre>\n"
```

Missing linebreak in several examples.
```suggestion
  example: 
  - "\n<pre><code>EventDataDialogDefinition def;\ndef.addCheckbox(&quot;checkme&quot;, &quot;My checkbox&quot;,true);</code></pre>\n"
```